AIPM-Registry/ETL_AND_QUERIES.md

```
@@ -0,0 +1,162 @@
+
+- `last_verified` timestam
+
+p
+
+- (Optional) `models` and `protocols` that the variant is bound to
+
+.
+
+--
+
+- #
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)